tests/pages/dashboard/ComponentsList-cy.js

```diff
@@ -2,7 +2,8 @@ describe('ComponentsList [10b]', function () {
 
   context('No Components Found [10f]', function () {
 
-    it('shows error when components not found [10d]', function () {
+    // TODO: Turn into unit test
+    xit('shows error when components not found [10d]', function () {
```

Comment:
The problem with that is that it is not an actual integration test then. We can easily test this as a unit test, but until we have something where we can actually change the cluster, this is going to be hard.

The whole point of having these tests is that they should work for actual clusters too